pkg/strategy/xgap/strategy.go

```diff
@@ -237,17 +237,28 @@ func (s *Strategy) CrossRun(ctx context.Context, _ bbgo.OrderExecutionRouter, se
 }
 
 func (s *Strategy) makeSpread(ctx context.Context, bestBid, bestAsk types.PriceVolume) error {
-	orderForms := []types.SubmitOrder{
-		{
-			Symbol:      s.Symbol,
-			Side:        types.SideTypeSell,
-			Type:        types.OrderTypeLimit,
-			Quantity:    fixedpoint.Max(bestBid.Volume, s.tradingMarket.MinQuantity),
-			Price:       bestBid.Price,
-			Market:      s.tradingMarket,
-			TimeInForce: types.TimeInForceIOC,
-		},
+	balances := s.tradingSession.GetAccount().Balances()
+	order := types.SubmitOrder{
+		Symbol:      s.Symbol,
+		Side:        types.SideTypeSell,
+		Type:        types.OrderTypeLimit,
+		Quantity:    fixedpoint.Max(bestBid.Volume, s.tradingMarket.MinQuantity),
+		Price:       bestBid.Price,
+		Market:      s.tradingMarket,
+		TimeInForce: types.TimeInForceIOC,
+	}
+	if err := tradingutil.HasSufficientBalance(
```

Comment:
HasSufficientBalance sounds you will return bool, but it returns error

Reply:
Fixed